invenio_app_ils/ui/backoffice/src/pages/ItemDetails/index.js

```diff
@@ -0,0 +1,18 @@
+import { connect } from 'react-redux';
+import ItemDetails from './ItemDetails';
+import { fetchItemDetails, postItemAction } from './state/actions';
+
+const mapDispatchToProps = dispatch => ({
+  fetchItemDetails: itemid => dispatch(fetchItemDetails(itemid)),
+  postItemAction: (url, data) => dispatch(postItemAction(url, data)),
```

Comment:
I am not sure there is an item action, is there any? Modifications to the items will go through record editor.

Reply:
That was part of copy-pasting, will remove it.